### ISSUE TITLE
Can't move latest log to backup folder

### ISSUE THREAD

**Issue by csaezl** [Norconex/collector-http#96]
Running the crawler in ```resume``` mode, I got the error and the run stopped:

![25-04-2015 20-00-37](https://cloud.githubusercontent.com/assets/10865115/7334050/3f438ec8-eb86-11e4-9116-aff5e2030622.jpg)

log..backup no contains the previous one
log..latest\log contains the previous one

progress..backup contains the previous one
progress..latest\status **is empty**, so JEF Monitor says the status is unkown

**Comment by OkkeKlein** [comment]
Looks like a permission problem to me.

**Comment by csaezl** [comment]
I run the collector twice before the third failed. In between nothing changed concerning folders nor permissions.

**Comment by csaezl** [comment]
And with Windows Explorer I can copy files to the tarjet folder

**Comment by csaezl** [comment]
I believe I have an idea about when it happens. I have a config file with 100 url to crawl. I run the config file just with 10 of them, just to test. The run ended. I deleted the 10 url and included another group of 20 url and run it with ```-a resume```. At the very beginning, I got the error reported.
I recovered the scenario after processing the first 10 URL and run the second 20 this time with ```-a start```. No error happened.
